src/common/utils.tsx

```diff
@@ -369,3 +370,20 @@ export const getContractIdFromTx = (tx?: Transaction | MempoolTransaction) => {
   if (tx.tx_type === 'smart_contract') contractId = tx.smart_contract.contract_id;
   return contractId;
 };
+
+export function initBigNumber(num: string | number | BigNumber) {
+  return BigNumber.isBigNumber(num) ? num : new BigNumber(num);
+}
+
+export const ftDecimals = (value: number | string | BigNumber, decimals: number = STX_DECIMALS) => {
+  const amount = initBigNumber(value);
+  return amount
```

Comment:
```suggestion
  return initBigNumber(value)
``` 

?

Reply:
I realized I copied these from the web wallet, so prob should be updated there too at some point.